### PR TITLE
Update dutch translation for v1.9.0

### PR DESCRIPTION
**Pull request by michaelarnauts**
### Check if this PR fulfills these requirements:
* [X] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [X] I have read the [**CONTRIBUTING**](../../master/Contributing.md) document.
* [X] I made sure there wasn't another [Pull Request](../../../pulls) opened for the same update/change
* [ ] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [X] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description

* I've included the changes that @prov3it made in https://github.com/CastagnaIT/plugin.video.netflix/pull/830
* Translated new strings
* Cleaned up some translations

@dagwieers you are maintainer of the nl-nl language, can you verify?

### CODE REVIEW

File: resources/language/resource.language.nl_nl/strings.po [hunk 1]
```diff
@@ -210,57 +210,57 @@ msgstr "Bibliotheek"
...
 
 msgctxt "#30055"
 msgid "Set for auto-selection"
-msgstr "Stel in voor automatische kiezen"
+msgstr "Automatisch kiezen van profiel activeren"
 
 msgctxt "#30056"
 msgid "Remove auto-selection"
-msgstr "Automatisch kiezen van profiel desactiveren"
+msgstr "Automatisch kiezen van profiel deactiveren"
```
Comment (dagwieers) [inline resources/language/resource.language.nl_nl/strings.po:233]: Beide zijn correct.

File: resources/language/resource.language.nl_nl/strings.po [hunk 2]
```diff
@@ -710,11 +710,11 @@ msgstr "Open de Up Next add-on instellingen"
 
 msgctxt "#30179"
 msgid "Show trailers & more"
-msgstr "Toon voorfilmpjes"
+msgstr "Toon trailers & meer"
```
Comment (dagwieers) [inline resources/language/resource.language.nl_nl/strings.po:713]: Never use ampersand ! Unless it is part of a name or title. Better use "en".
BTW This also applies to English.

**Review by dagwieers — APPROVED** [review]
Fine by me. I was already halfway, but I think your translation is generally better than mine.
I never use software in Dutch as it mostly feels awkward to me, so I don't mind if someone else wants to take over :wink:

File: resources/language/resource.language.nl_nl/strings.po [hunk 3]
```diff
@@ -22,7 +22,7 @@ msgstr "Netflix"
 
 msgctxt "Addon Description"
 msgid "Netflix VOD Services Add-on"
-msgstr "Bekijk Netflix films en tvprogramma's in Kodi."
+msgstr "Netflix films en tv-programma's"
```
Comment (dagwieers) [inline resources/language/resource.language.nl_nl/strings.po:25]: I think the English original should be improved at this point.

**Comment by michaelarnauts** [comment]
I have an PR open for suggestions for the English translation: https://github.com/CastagnaIT/plugin.video.netflix/pull/852

File: resources/language/resource.language.nl_nl/strings.po (hunk 1 above)
Comment (michaelarnauts) [reply resources/language/resource.language.nl_nl/strings.po:233]: Ha, blijkbaar, wist ik niet (deze change was van de oorspronkelijke PR), maar ik gebruik zelf ook meer deactiveren, en deze komt ook veel meer voor op Google:

"desactiveren": 25600 hits
"deactiveren": 633000 hits

Lijkt me oké om voor `deactiveren` te gaan dus?

**Comment by michaelarnauts** [comment]
Okay, so these changes can be merged. @CastagnaIT you can add me to the Dutch list.

**Comment by CastagnaIT** [comment]
well

File: resources/language/resource.language.nl_nl/strings.po (hunk 1 above)
Comment (dagwieers) [reply resources/language/resource.language.nl_nl/strings.po:233]: Sure.

**Pull request merged**